### DISCUSSION TITLE
Add support for non-persistent Embedded properties in entities

### DISCUSSION THREAD

**Discussion by veselin-angelov** [mikro-orm/mikro-orm#5554] [Ideas]
**Is your feature request related to a problem? Please describe.**
As of now MikroORM allows for non-persistent properties in entities and I am trying to achieve the same with an Embedded, which is not supported. I find non-persistent properties very useful and don't see why it is not allowed for Embedded types also.

**Describe the solution you'd like**
```ts
@Embedded({
  entity: () => SomeEmbeddable,
  prefix: 'some_prefix_',
  persist: false,
})
public someEmbeddable: SomeEmbeddable;
```

**Describe alternatives you've considered**
I have considered, just deleting the columns from the database, which works, but on each migration is generated again.
Also I have considered just in-lining the properties with `persist: false` on each, but I want them to be in an object.

**Additional context**
Cannot think of anything else right now.

**Comment by B4nan** [comment]
I am not entirely sure what you are proposing here, embedded property like this won't be persisted, only the "child properties" of the target embeddable entity will be. If you see something else, provide a complete reproduction.
